src/grantRegistry.ts

```diff
@@ -44,10 +47,13 @@ export function handleGrantUpdated(event: GrantUpdated): void {
     entity.id = event.params.id.toHex()
     entity.owner = event.params.owner
     entity.payee = event.params.payee
-    entity.metaPtr = event.params.metaPtr
+
+    const metaPtr = event.params.metaPtr
+    entity.metaPtr = [metaPtr.protocol.toString(), metaPtr.pointer].join('-')
+
     entity.lastUpdatedBlockNumber = event.block.number
     entity.lastUpdatedTimestamp = event.block.timestamp
-    entity.createdAtTimestamp = 
+    entity.createdAtTimestamp =
         entity.createdAtTimestamp.toString() !== "0" ? entity.createdAtTimestamp : event.block.timestamp
```

Comment:
Why do we need this conditional? `createdAt` is a storage param that gets set once at creation and never changes,  so I'm not sure how that works with the graph

Reply:
Ahh, if I remember correctly, my local environment was filled with 0's and I couldn't work out where they were coming from so I conditionally set it to the blocks timestamp so that I could see some timestamps to test my queries. 

There is a good chance we don't need this.

Reply:
> There is a good chance we don't need this.

Should we validate that, and if correct, remove it then?

Reply:
Okay, I was wrong - when we load the entity for the first time the `createdAtTimestamp` will be empty (on the entity) so we set it to the events block timestamp, the second time that the entity is loaded (eg on `handleGrantUpdated()`), we keep the original value that we set on the first run.

For the `GrantRegistry` we can change this to use `event.params.time` but it should be the same if we pull it from the block or from the emitted event log (it will be pointing to the same block). We're not able to do the same for the `GrantRound`s because we don't emit a timestamp on `GrantRoundCreated`.

I don't think any changes are required here.

Reply:
Makes sense, thanks!